web/controllers/auth.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Notox.Auth do
+  import Plug.Conn
+
+  def init(_opts) do
+  end
+
+  defp get_current_user(conn) do
+    current_user_id = get_session(conn, :user_id)
+    current_user_id && Notox.Repo.get(Notox.User, current_user_id)
```

Comment:
The intent of this function is to return a user object, not a boolean. Despite adding extra lines, I think it would be more clear to use a conditional here for control flow.